src/nupic/engine/RegionImplFactory.cpp

```diff
@@ -263,26 +244,10 @@ static RegionImpl * deserializePyNode(DynamicPythonLibrary * pyLib,
     std::string fullNodeType = std::string(package) + std::string(".") +
                                std::string(nodeType.c_str() + 3);
 
-    // Check if node exists and continue if not
-    std::string nodePath = Path::join(getPackageDir(pyLib->getRootDir(), package), 
-           std::string(nodeType.c_str() + 3) + std::string(".py"));
-
-    if (!Path::exists(nodePath))
```

Comment:
How about use this default path when user doesn't inform the full path and then when it really doesn't exist then return null? This way we prevent some extra code.